### PR TITLE
Minor fixes and runtime raises on init

### PR DESCRIPTION
**Pull request by mmphego**

### CODE REVIEW

File: mkat_fpga_tests/__init__.py [hunk 1]
```diff
@@ -126,16 +126,10 @@ def katcp_rct(self):
         if self._katcp_rct is None:
             try:
                 self.katcp_array_port = int(
-                self.rct.req.array_list()[1][0].arguments[1])
+                    self.rct.req.array_list()[1][0].arguments[1])
```
Comment (brickZA) [inline mkat_fpga_tests/__init__.py]: Too many indents?

File: mkat_fpga_tests/__init__.py [hunk 2]
```diff
@@ -126,16 +126,10 @@ def katcp_rct(self):
...
             except IndexError:
-                print 'Index error at line 126\n\n\n'
-                import IPython;IPython.embed()
-                #self.start_correlator()
-                # multicast_ip = self.corr_conf['test_confs']['source_mcast_ips']
-                #self.rct.req.array_assign('array0',
-                #        *multicast_ip.split(','))
-                #self.katcp_array_port = int(
-                #self.rct.req.array_list()[1][0].arguments[1])
+                LOGGER.error('Failed to assign katcp array port number.')
```
Comment (brickZA) [inline mkat_fpga_tests/__init__.py]: No need to log if you are raising an exception?

File: mkat_fpga_tests/__init__.py [hunk 3]
```diff
@@ -174,7 +168,8 @@ def start_correlator(self, retries=30, loglevel='INFO'):
                 self.array_number = array_list_messages[0].arguments[0]
                 self.rct.req.array_halt(self.array_number)
         except IndexError:
-            LOGGER.info ("Already cleared array")
+            LOGGER.error("Unable to halt array due to empty array number")
```
Comment (brickZA) [inline mkat_fpga_tests/__init__.py]: No need to log if you are also raising an exception?

File: mkat_fpga_tests/__init__.py (hunk 1 above)
Comment (mmphego) [inline mkat_fpga_tests/__init__.py]: Line was previously not indented.

**Comment by mmphego** [comment]
Removed loggers and fixed the indentation.
Ready to merge

File: mkat_fpga_tests/__init__.py (hunk 1 above)
Comment (brickZA) [inline mkat_fpga_tests/__init__.py]: Oh, I see, thre is a bracket, right you are :)

**Pull request merged**